docs/administration/managing-users-and-teams/index.md

```diff
@@ -40,14 +40,33 @@ Invite codes are only valid for 48-hours aftering being generated, so make sure
 
 ## Creating teams {#Managingusersandteams-Creatingteams}
 
-Creating teams lets you assign the same roles to groups of users. Users can be added to or removed from multiple teams, making it eaiser to manage permissions for specific users and teams. You can create new teams by using the **Add Team** button.
+Creating teams lets you assign the same roles to groups of users. Users can be added to or removed from multiple teams, making it easier to manage permissions for specific users and teams. You can create new teams by using the **Add Team** button.
 
 For example, we can create a team that gives Anne and Bob access to view projects and deploy them to pre-production environments by assigning the role **Project deployer** to the team.
 
 We limit which projects and environments these permissions apply to by adding specific projects and environments to the team.
 
 ![](teams-create-new.png "width=500")
 
+### Locking down which projects you can see
+
+When specifying both `Project Groups`and `Projects`  filters please be aware that both these filters complement each other, to better illustrate these filters in action lets consider the following project structure:
```

Comment:
When specifying both `Project Groups` and `Projects` filters please be aware that both these filters complement each other. To better illustrate these filters in action let's consider the following project structure: